src/storage/memory_cache/write_back_task.rs

```diff
@@ -112,15 +141,34 @@ where
     /// Flush a pending block to the backend
     async fn flush_a_block(&mut self) {
         if let Some(BlockCoordinate(ino, block_id)) = self.lru_queue.pop_front() {
-            if let Some((block, sender)) = self
+            if let Some((block, tx)) = self
                 .pending_blocks
                 .get_mut(&ino)
                 .and_then(|file_level_pending_blocks| file_level_pending_blocks.remove(&block_id))
             {
-                let res = self.storage.backend().store(ino, block_id, block).await;
-                sender.send(res).unwrap_or_else(|res| {
-                    error!("The receiver is closed unexpectedly, with storage result: {res:?}");
-                });
+                if self
+                    .block_flush_spawn_handle
+                    .spawn(|_| {
+                        write_back_one_block(
+                            Arc::clone(&self.storage),
+                            ino,
+                            block_id,
+                            block.clone(),
+                            tx,
+                        )
+                    })
+                    .await
+                    .is_err()
+                {
+                    // Ensure that the block is flushed to the backend, and notify the loop to
+                    // shutdown.
+                    error!("Try to spawn a `BlockFlush` task after shutdown.");
+                    if let Err(e) = self.storage.backend().store(ino, block_id, block).await {
```

Comment:
When shutdown ,just return directly.

Reply:
Print the error in log, and ignore the un-flushed block?